src/rfbzero/redox_flow_cell.py

```diff
@@ -142,12 +159,16 @@ def __init__(self, cls_volume: float, ncls_volume: float, cls_start_c_ox: float,
         if not isinstance(self.n_cls, int) or not isinstance(self.n_ncls, int):
             raise ValueError("'n_cls' and 'n_ncls' must be integers")
 
-        if self.cls_volume >= self.ncls_volume:
-            raise ValueError("'cls_volume' must be < 'ncls_volume'")
+        if self.init_ocv == 0.0 and self.cls_volume >= self.ncls_volume:
+            raise ValueError("'cls_volume' must be < 'ncls_volume' in a symmetric cell")
 
         if self.init_ocv == 0.0 and self.n_cls != self.n_ncls:
             raise ValueError("Symmetric cell (0 volt OCV) requires 'n_cls' and 'n_ncls' to be equal (same species)")
 
+        if (self.cls_volume * self.n_cls * (self.c_ox_cls + self.c_red_cls)
+                >= self.ncls_volume * self.n_ncls * (self.c_ox_ncls + self.c_red_ncls)):
+            raise ValueError("Initial capacity of CLS must be less than initial capacity of NCLS")
```

Comment:
Thanks for adding a check. Maybe add two local variables before for init_cls_capacity and init_ncls_capacity to make the if statement more readable

Reply:
done